api/all/src/main/java/io/opentelemetry/api/internal/TemporaryBuffers.java

```diff
@@ -12,7 +12,7 @@
  * should get a temporary buffer, fill it with data, and finish by converting into the derived
  * object within the same method to avoid multiple usages of the same buffer.
  */
-final class TemporaryBuffers {
+public final class TemporaryBuffers {
```

Comment:
We should add the boilerplate about internal classes to this javadoc, I think.

Reply:
Yup added